### DISCUSSION TITLE
Consulta, tengo que crear una  llave SSH dentro de la configuración de git?

### DISCUSSION THREAD

**Discussion by dulcepiciana** [INGCOM-UNRN-P1/cursada-2023#22] [Preguntas y respuestas] [answered]
Si es asi, también deberia cargarla públicamente?

**Comment by martinvilu** [comment]
Si, es necesario crear la llave SSH, y cargar en github la parte pública de la misma.

Estas siguiendo el apunte de este tema?

**Comment by JeanFranco99** [comment, marked as answer]
Si.
Tenes que crear la llave SSH poniendo _cat ~/.ssh/id_rsa.pub_
Eso te va a tirar la key que vas a tener que pegar [aca](https://github.com/settings/keys), creando una nueva SSH key

**Reply by martinvilu** [reply]
@dulcepiciana No olvides marcar la solucion si te dio resultado. Lo marqué como respondido ¿pero _te funciono_?

**Reply by dulcepiciana** [reply]
Si! Muchas gracias
